gogs/files/templates/gogs-persistent-template.yaml

```diff
@@ -0,0 +1,383 @@
+kind: Template
+apiVersion: v1
+metadata:
+  annotations:
+    description: The Gogs git server (https://gogs.io/)
+    tags: instant-app,gogs,go,golang
+  name: gogs
+objects:
+- kind: SecurityContextConstraints
+  metadata:
+    annotations:
+      kubernetes.io/description: anyuid provides all features of the restricted SCC
+        but allows users to run with any UID and any GID.
+    name: anyuid
+  users:
+  - system:serviceaccount:gogs:${APPLICATION_NAME}
```

Comment:
Can you add a section to README stating what type of role/privileges are required in order for the applier to run? If run as a regular user, the applier will fail on getting/creating SCCs.

Reply:
Done